### PR TITLE
mumbai testnet deployment settings

### PR DESCRIPTION
**Pull request by asimpk**
tested mumbai testnet deployment using frontend sdk

### CODE REVIEW

File: packages/lib-ethers/src/EthersKumoConnection.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import kovan from "../deployments/kovan.json";
 import rinkeby from "../deployments/rinkeby.json";
```
Comment (udosson) [inline packages/lib-ethers/src/EthersKumoConnection.ts:9]: Could you remove the old artifacts from Liquity? We only need Mumbai and Polygon

File: packages/dev-frontend/src/hooks/KumoContext.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ const wsParams = (network: string, infuraApiKey: string): [string, string] => [
   network
 ];
 
-const supportedNetworks = ["homestead", "kovan", "rinkeby", "ropsten", "goerli"];
+const supportedNetworks = ["homestead", "kovan", "rinkeby", "ropsten", "goerli", "mumbai"];
```
Comment (udosson) [inline packages/dev-frontend/src/hooks/KumoContext.tsx:37]: Could you remove the old artifacts from Liquity? We only need Mumbai and Polygon

**Review by udosson — CHANGES_REQUESTED** [review]
@asimpk - thanks for the PR. Could you (1) please remove all not used artifacts related to the Ethereum mainnet and its testnets, please? And (2) could you please synchronize with @arthurka-o regarding the deployment? Thank you!

**Comment by asimpk** [comment]
> @asimpk could you change the base branch to `kumodao:30-ll-deploy-the-protocol-to-mumbai-polygon-testnet`, please?

@udosson yeah sure thanks, PR will be ready for merge after my session with Arthur

**Comment by udosson** [comment]
> > @asimpk could you change the base branch to `kumodao:30-ll-deploy-the-protocol-to-mumbai-polygon-testnet`, please?
> 
> @udosson yeah sure thanks, PR will be ready for merge after my session with @arthurka-o

perfect, @asimpk. Please, take care of the commits that are part of your PR. I recommend to have only 1 commit with only the changes needed rather than 62 in total. Thank you

**Comment by asimpk** [comment]
> > > @asimpk could you change the base branch to `kumodao:30-ll-deploy-the-protocol-to-mumbai-polygon-testnet`, please?
> > 
> > 
> > @udosson yeah sure thanks, PR will be ready for merge after my session with @arthurka-o
> 
> perfect, @asimpk. Please, take care of the commits that are part of your PR. I recommend to have only 1 commit with only the changes needed rather than 62 in total. Thank you



> > > @asimpk could you change the base branch to `kumodao:30-ll-deploy-the-protocol-to-mumbai-polygon-testnet`, please?
> > 
> > 
> > @udosson yeah sure thanks, PR will be ready for merge after my session with @arthurka-o
> 
> perfect, @asimpk. Please, take care of the commits that are part of your PR. I recommend to have only 1 commit with only the changes needed rather than 62 in total. Thank you

hi @udosson tbh I don't know why I'm getting list of all these commits , these are all old, I make only 3 commits first one two days ago and then yesterday, It seems some branch issues or maybe something else

**Comment by udosson** [comment]
How I would proceed:
```git checkout main
# if you haven't added the upstream repository, do: git remote add upstream https://github.com/kumodao/borrowprot.git
git fetch upstream
git merge upstream/main```

Now, your local `main` branch of your fork is updated with `main` from kumodao.

Next, create a new branch from `main` by running `git checkout -b NAME_OF_YOUR_BRANCH` and `cherry-pick` your commit or do the changes again.

Let me know if I can support you further

**Comment by udosson** [comment]
Merged similar PR #133

**Pull request closed without merging**